azure/packages/azure-client/src/AzureAudience.ts

```diff
@@ -14,6 +14,7 @@ import { type AzureMember, type AzureUser, type IAzureAudience } from "./interfa
  * @remarks Operates in terms of {@link AzureMember}s.
  *
  * @public
+ * @deprecated - class will no longer be exposed. use IAzureAudience instead
```

Comment:
Nit
```suggestion
 * @deprecated This class will no longer be exposed. use {@link IAzureAudience} instead
```

`@deprecated` comments don't want a `-` before their contents. Only `@param` and `@typeParam` want them.